fulp_modules/safety_moth/safety_moth.dm

```diff
@@ -0,0 +1,49 @@
+/datum/ert/safety_moth
+    mobtype = /mob/living/carbon/human/species/moth
+    leader_role = /datum/antagonist/ert/safety_moth
+    enforce_human = FALSE
+    roles = /datum/antagonist/ert/safety_moth
+    mission = "Ensure that proper safety protocols are being followed by the crew."
+    teamsize = 1
+    polldesc = "an experienced Nanotrasen engineering expert"
+    opendoors = FALSE
+
+
+/datum/antagonist/ert/safety_moth
+    name = "Safety Moth"
+    outfit = /datum/outfit/centcom/ert/engineer/safety_moth
+    role = "Safety Moth"
+
+/datum/antagonist/ert/safety_moth/on_gain()
+    forge_objectives()
+    . = ..()

```

Comment:
This should be at the end I think, someone reported safety moths not being pacifists, that might be why